src/linear_algebra/finite_dimensional.lean

```diff
@@ -850,6 +850,72 @@ by { unfold findim, simp [dim_top] }
 
 end top
 
+lemma eq_zero_of_findim_eq_zero [finite_dimensional K V]
+  (hV : findim K V = 0) (x : V) : x = 0 :=
+begin
+  obtain ⟨B, hB⟩ := exists_is_basis_finset K V,
+  have := (findim_eq_card_finset_basis hB).symm,
+  rw [hV, finset.card_eq_zero, finset.eq_empty_iff_forall_not_mem] at this,
+  rw ← is_basis.equiv_fun_total hB x,
+  convert finset.sum_empty,
+  rw finset.univ_eq_empty,
+  rintro ⟨b, hB⟩,
+  exact this b hB,
+end
+
+lemma subsingleton.is_basis (h : subsingleton V) : is_basis K (λ x : fin 0, (0 : V)) :=
+begin
+  refine is_basis_empty (finset.univ_eq_empty.mp rfl) _,
+  intro x,
+  rw [subsingleton.elim x 0],
+end
```

Comment:
This needs a rebase (and probably removing?) now that #6281  is merged

Reply:
Thanks for the reminder